testing/generate.js

```diff
@@ -212,12 +212,19 @@ function main() {
   
   // TODO: Support protected applications
   const protectedPath = path.join(basePath, '.retool', 'protected-apps.yaml');
-  if (fs.existsSync(path)) {
+  if (fs.existsSync(protectedPath)) {
```

Comment:
@mokshjawa  this won't work for Gitlab (file does not exist) - but we can patch after

Reply:
https://linear.app/retool/issue/DTC-2230/protected-apps-check-in-ci-will-not-work-for-gitlab